cfg/Instructions.cc

```diff
@@ -196,14 +196,15 @@ string TAbsurd::showRaw(const core::GlobalState &gs, int tabs) const {
 }
 
 string VariableUseSite::toString(const core::GlobalState &gs) const {
-    if (this->type) {
+    if (this->variable == core::LocalVariable::unconditional() || this->type == nullptr) {
+        return this->variable.toString(gs);
+    } else {
         return fmt::format("{}: {}", this->variable.toString(gs), this->type->show(gs));
     }
-    return this->variable.toString(gs);
 }
 
 string VariableUseSite::showRaw(const core::GlobalState &gs, int tabs) const {
-    if (this->type == nullptr) {
+    if (this->variable == core::LocalVariable::unconditional() || this->type == nullptr) {
```

Comment:
This change to use a name that exists revealed that the `VariableUseSite` for the `BlockExit::cond` did have a type: `NilClass` (because it was uninitialized). We had always hidden that in printing, so I changed our printing code to reflect that explicitly rather than make a large diff to the snapshots.